### PR TITLE
SPCTR-3524 Global - Resolved an issue wherein the custom-template/wooCommerce product cat/taxonomy styles would not load as intended in FSE theme.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3524

### Screenshots

- https://d.pr/v/pmtUkB

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the task

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,6 +221,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Fix: Tabs - Resolved an issue in which the tabs wrapper margin would appear differently in the editor and the front-end.
 * Fix: Icon List - Resolved an issue wherein the vertical layout alignment would not appear as intended in the editor's tablet view. 
 * Fix: FAQ - Resolved an issue wherein the block content padding would appear differently in the editor and the front-end.
+* Fix: Global - Resolved an issue of custom template style not loading issue in FSE theme.
```
Comment (bsf-zanev) [inline README.md:224]:
```suggestion
* Fix: Global - Resolved an issue wherein the custom template-styles would not load as intended in FSE theme.
```
OR
```suggestion
*Fix: Global - Resolved an issue wherein the WooCommerce and custom template-styles would not load as intended in FSE theme.
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, just a changelog update required.

File: README.md (hunk 1 above)
Comment (AbhijitNage123) [reply README.md]: Done

**Review by bsf-zanev — APPROVED** [review]

File: classes/class-uagb-post-assets.php [hunk 2]
```diff
@@ -274,10 +280,18 @@ public function get_woocommerce_template() {
...
 	 * @return string The determined post type.
 	 */
-	private function determine_template_post_type() {
+	private function determine_template_post_type( $post_id ) {
+		// Check if post id is passed.
+		if ( ! empty( $post_id ) ) {
+			$template_slug = get_page_template_slug( $post_id );
+			if ( ! empty( $template_slug ) ) {
+				return $template_slug;
+			}
```
Comment (adi3890) [inline classes/class-uagb-post-assets.php:293]: As tempalte slug can be false also so do we need to mamange that case too.

Comment (AbhijitNage123) [reply classes/class-uagb-post-assets.php:293]: If false it will not go into the if loop and run further code that written after if loop closed.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**